File: util/sms/src/main/java/top/bettercode/sms/aliyun/AliSmsProperties.java
package top.bettercode.sms.aliyun;

import org.springframework.boot.context.properties.ConfigurationProperties;

@ConfigurationProperties(prefix = "summer.sms.aliyun")
public class AliSmsProperties {

  private String url = "https://dysmsapi.aliyuncs.com";
  private String accessKeyId;
  private String accessKeySecret;
  private String regionId = "cn-chengdu";
  /**
   * 请求连接超时时间毫秒数
   */
  private int connectTimeout = 10000;
  /**
   * 请求读取超时时间毫秒数
   */
  private int readTimeout = 10000;

  //--------------------------------------------


  public String getUrl() {
    return url;
  }

  public void setUrl(String url) {
    this.url = url;
  }

  public String getAccessKeyId() {
    return accessKeyId;
  }

  public void setAccessKeyId(String accessKeyId) {
    this.accessKeyId = accessKeyId;
  }

  public String getAccessKeySecret() {
    return accessKeySecret;
  }

  public void setAccessKeySecret(String accessKeySecret) {
    this.accessKeySecret = accessKeySecret;
  }

  public String getRegionId() {
    return regionId;
  }

  public void setRegionId(String regionId) {
    this.regionId = regionId;
  }

  public int getConnectTimeout() {
    return connectTimeout;
  }

  public void setConnectTimeout(int connectTimeout) {
    this.connectTimeout = connectTimeout;
  }

  public int getReadTimeout() {
    return readTimeout;
  }

  public void setReadTimeout(int readTimeout) {
    this.readTimeout = readTimeout;
  }
}

File: tools/sms/src/main/kotlin/top/bettercode/summer/tools/sms/aliyun/AliSmsProperties.kt
package top.bettercode.summer.tools.sms.aliyun

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "summer.sms.aliyun")
class AliSmsProperties {
    //--------------------------------------------
    var url = "https://dysmsapi.aliyuncs.com"
    var accessKeyId: String? = null
    var accessKeySecret: String? = null
    var regionId = "cn-chengdu"

    /**
     * 模拟发送
     */
    var mock = false

    /**
     * 请求连接超时时间毫秒数
     */
    var connectTimeout = 10000

    /**
     * 请求读取超时时间毫秒数
     */
    var readTimeout = 10000
}
